templates/pages/signup.jsx

```diff
@@ -79,6 +80,7 @@ var Signup = React.createClass({
 
   handleFormData: function(error, data) {
     if ( error ) {
+      ga.event({category: 'Signup', action: 'Error during form validation'});
```

Comment:
Also, Ignore this comment. As I see we are doing this in `templates/components/form/form.jsx`